examples/apigee-x-vpn-hub-and-spoke-design/variables.tf

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "gcp_org_id" {
+  description = "Organization ID that the projects are created under"
+  type        = string
+}
+
+variable "gcp_billing_id" {
+  description = "Billing ID that is attached to the project"
+  type        = string
+}
+
+variable "cidr_mask" {
+  description = "The CIDR mask for creating Apigee instance"
+  type        = number
+}
+ 
+variable "subnet_1" {
+  description = "The region for a subnet"
+  type        = string
+}
+
+variable "subnet_2" {
```

Comment:
If this is an example you can provide defaults for these rather than requiring inputs